vendor/bitcoin_witness/src/primed_transaction.rs

```diff
@@ -176,7 +187,7 @@ mod test {
         let rate = 42.0;
 
         let estimated_weight = primed_txn.estimate_weight();
-        let transaction = primed_txn.sign_with_rate(rate);
+        let transaction = primed_txn.sign_with_rate(rate).unwrap();
```

Comment:
🤡 if you make this test function return a result you can use `?` instead of unwrap.